services/cert-manager/1.12.2/priorityclass-resource-quota/resourcequota.yaml

```diff
@@ -1,14 +1,15 @@
 apiVersion: v1
 kind: ResourceQuota
 metadata:
-  name: fluent-bit-critical-pods
-  namespace: ${releaseNamespace}
+  name: cert-manager-critical-pods
+  namespace: cert-manager
 spec:
   hard:
-    pods: "1G"
```

Comment:
should we keep `1G` (a large number) since this isn't configurable? 🤔

i doubt we'd ever need to change it since i don't think cert-manager scales horizontally or anything

Reply:
yes! now that we know what 1G is (probably)